src/main/java/firebase/sdk/dashboard/api/SDKResource.java

```diff
@@ -25,10 +27,11 @@
 @Consumes(MediaType.APPLICATION_JSON)
 public class SDKResource {
 
-  private String platform;
+  private Platform platform;
+  private final SDKDao SDKDAO = new SDKDaoDatastore();
```

Comment:
Read about private final vs private static final in java. See, whats relevant here.

Reply:
Ah I see. private static final makes more sense. Thank you!